app/models/house.rb

```diff
@@ -3,10 +3,27 @@ class House < ApplicationRecord
   has_many :home_owners_associations, through: :hood
   has_one :house_metadatum, dependent: :destroy
 
+  delegate :zpid, to: :house_metadatum, allow_nil: true
+
+  after_initialize :set_price_history
```

Comment:
Ensures we have a default empty hash on `House.new` or retrieving an instance of a house from the database that doesn't have a price history.